tests/spec/w3c/headers-spec.js

```diff
@@ -557,6 +557,105 @@ describe("W3C — Headers", () => {
     });
   });
 
+  describe("valid level config", () => {
+    it("updates relevant locations with the level when level is the integer 0", async () => {
+      const body = `
+        <h1 id="title">Spec <code>Marked</code> Up</h1>${makeDefaultBody()}`;
+      const ops = makeStandardOps({}, body);
+      const newProps = { level: 0, specStatus: "REC", shortName: "abc" };
```

Comment:
We can pass newProps as first argument to makeStandardOps

Reply:
``` js
const ops = makeStandardOps({ level: 0, specStatus: "REC", shortName: "abc" }, body);
```

Reply:
nice! fixed it for all the specs.